### ISSUE TITLE
Enhance a11y for `Accordion`

### ISSUE THREAD

**Issue by hirotomoyamada** [yamada-ui/yamada-ui#5393]
name: Accordion

### Description
3 accessibility violation was found in the `Accordion`.

#### [Components / Accordion / Variant](https://yamada-ui.github.io/yamada-ui/?path=/story/components-accordion--variant)

Received:
Landmarks should have a unique role or role/label/title (i.e. accessible name) combination (landmark-unique)

Fix any of the following:
  The landmark must have a unique aria-label, aria-labelledby, or title to make landmarks distinguishable

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/landmark-unique?application=axeAPI

#### [Components / Accordion / Variant](https://yamada-ui.github.io/yamada-ui/?path=/story/components-accordion--variant)

Received:
Landmarks should have a unique role or role/label/title (i.e. accessible name) combination (landmark-unique)

Fix any of the following:
  The landmark must have a unique aria-label, aria-labelledby, or title to make landmarks distinguishable

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/landmark-unique?application=axeAPI

#### [Components / Accordion / Variant](https://yamada-ui.github.io/yamada-ui/?path=/story/components-accordion--variant)

Received:
Landmarks should have a unique role or role/label/title (i.e. accessible name) combination (landmark-unique)

Fix any of the following:
  The landmark must have a unique aria-label, aria-labelledby, or title to make landmarks distinguishable

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/landmark-unique?application=axeAPI

### Additional Information
While it's ideal to cover all tests, if there are too many lines to work on, please create a PR within your capacity.
Any deficiencies not covered by the PR will be automatically created as issues.
